README.md

```diff
@@ -5,46 +5,196 @@
 ![Azure DevOps builds (branch)](https://img.shields.io/azure-devops/build/openvinoci/b2bab62f-ab2f-4871-a538-86ea1be7d20f/13?label=Public%20CI)
 [![PyPI Downloads](https://pepy.tech/badge/openvino)](https://pepy.tech/project/openvino)
 
-This toolkit allows developers to deploy pre-trained deep learning models
-through a high-level OpenVINO™ Runtime C++ and Python APIs integrated with application logic.
+## Contents:
+
+ - [What is OpenVINO?](#what-is-openvino-toolkit)
+    - [Components](#components)
+ - [Supported Hardware matrix](#supported-hardware-matrix)
+ - [License](#license)
+ - [Documentation](#documentation)
+ - [Tutorials](#tutorials)
+ - [Products which use OpenVINO](#products-which-use-openvino)
+ - [System requirements](#system-requirements)
+ - [How to build](#how-to-build)
+ - [How to contribute](#how-to-contribute)
+ - [Get a support](#get-a-support)
+ - [See also](#see-also)
+
+## What is OpenVINO toolkit?
+
+OpenVINO™ toolkit allows developers to deploy pre-trained deep learning models through a high-level OpenVINO™ Runtime C++ and Python APIs integrated with application logic.
```

Comment:
`What is OpenVINO toolkit` is the section for newcomers, who has either beginner or intermediate level knowledge about openvino. In my opinion, this section is one of the most vital sections to attract their attention to look further at the repo to know about the killer features.

It would be more appealing if lots of visuals are presented here rather than providing more text. This way the reader wouldn't get lost in the mechanics and the details of the library, but rather would see what openvino could achieve. 

It would be possible to use visuals from Jupyter Notebooks, for instance, and attach them here as GIFs. This way, the developer/reader's attention could be attracted easier.

> A picture is worth a thousand words

Reply:
I will create a ticket to add new images which will show OpenVINO benefits.